### PR TITLE
xdotool: added more examples

### PR DESCRIPTION
**Pull request by jimfilippou**
I have added some more examples to xdotool since i didn't find them in there. Examples include sending text & keystrokes as well as focusing the current window and acquiring the current's window id

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/xdotool.md [hunk 1]
```diff
@@ -9,3 +9,19 @@
...
+
+- Focus on window with the id of 12345:
+
+`xdotool windowfocus --sync 12345`
+
+- Send a Hello World message with 500ms delay for each letter:
+
+`xdotool type --delay 500 "Hello World"`
+
+- Press enter:
```
Comment (sbrl) [inline pages/linux/xdotool.md]: Perhaps being more verbose with "Press the enter key" would aid clarity here?

File: pages/linux/xdotool.md [hunk 2]
```diff
@@ -9,3 +9,19 @@
...
+
+- Get the id of the currently active window:
+
+`xdotool getactivewindow`
+
+- Focus on window with the id of 12345:
+
+`xdotool windowfocus --sync 12345`
+
+- Send a Hello World message with 500ms delay for each letter:
```
Comment (sbrl) [inline pages/linux/xdotool.md]: Not sure that `Hello World` needs to be in title case here.

Also, perhaps `message, with a 500ms` would read better?

File: pages/linux/xdotool.md [hunk 3]
```diff
@@ -9,3 +9,19 @@
 - Click the right mouse button:
 
 `xdotool click {{3}}`
+
+- Get the id of the currently active window:
+
+`xdotool getactivewindow`
+
+- Focus on window with the id of 12345:
```
Comment (sbrl) [inline pages/linux/xdotool.md]: Perhaps `Focus on the window with` would read better here?

**Comment by jimfilippou** [comment]
Thanks for pointing out these recommendations, my english is not perfect 😆

**Pull request merged**